src/style.css

```
@@ -0,0 +1,78 @@
+html {
+  margin: 0;
+}
+
+body {
+  margin: 0;
+}
+
+header {
+  align-items: center;
+  margin: 0 50px;
+  display: flex;
+  justify-content: space-between;
+}
+
+img {
+  height: 40px;
+  width: 40px;
+}
+
+.navigation {
+  display: flex;
+  height: 60px;
+  margin: 0;
+  padding: 0;
+}
+
+.nav_item {
+  margin: 0 10px;
+}
+
+.nav_item:first-of-type {
+  margin-left: 0;
+}
+
+.nav_item:last-of-type {
+  margin-right: 0;
+}
+
+.nav_link {
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  height: 60px;
+  font-size: 12px;
+  line-height: 14px;
+  color: rgba(0, 0, 0, 1);
+}
+
+.nav_link:hover {
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  height: 60px;
+  font-size: 12px;
+  line-height: 14px;
+  color: rgba(0, 172, 220, 1);

```

```suggestion
  color: rgba(0, 172, 220, 1);
```

ти вказав ці всі стилі вище для чого їх тут повторювати